flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/api/TableEnvironmentITCase.scala

```diff
@@ -429,6 +429,26 @@ class TableEnvironmentITCase(tableEnvName: String, isStreaming: Boolean) extends
     assertLastValues(sink2Path)
   }
 
+  @Test
+  def testExecutionPlanFromStatementSet(): Unit = {
+    val sink1Path = TestTableSourceSinks.createCsvTemporarySinkTable(
+      tEnv, new TableSchema(Array("first"), Array(STRING)), "MySink1")
+
+    val sink2Path = TestTableSourceSinks.createCsvTemporarySinkTable(
+      tEnv, new TableSchema(Array("last"), Array(STRING)), "MySink2")
+
+    val stmtSet = tEnv.createStatementSet()
+    stmtSet.addInsert("MySink1", tEnv.sqlQuery("select first from MyTable"))
+        .addInsertSql("insert into MySink2 select last from MyTable")
+
+    val actual = stmtSet.explain(ExplainDetail.JSON_EXECUTION_PLAN)
+    val expected =
+      TableTestUtil.readFromResource("/explain/testExecutionPlanFromStatementSet.out")
+
+    assertEquals(replaceStreamNodeIdAndParallelism(expected),
+      replaceStreamNodeIdAndParallelism(actual))
```

Comment:
This test case is a unit test, not an integration test, so this test case should be removed to `TableEnvironmentTest`

nit: `sink1Path` and `sink2Path` are unnecessary

Reply:
You means that move this tests to `TableEnvironmentTest ` from `TableEnvironmentITCase`?

Reply:
yes, sorry for the typo word "removed"

Reply:
Okay